docs/source/matplotlib/gridliner.rst

```diff
@@ -41,8 +61,9 @@ class to produce customized gridlines and tick labels:
     gl.left_labels = False
     gl.xlines = False
     gl.xlocator = mticker.FixedLocator([-180, -45, 0, 45, 180])
-    gl.xformatter = LONGITUDE_FORMATTER
-    gl.yformatter = LATITUDE_FORMATTER
+    gl.ylocator = LatitudeLocator()
+    gl.xformatter = LongitudeFormatter
```

Comment:
Should this also be creating an instance?

Reply:
You are right. Just a mistake.